### ISSUE TITLE
remove delete image button from image upload page

### ISSUE THREAD

**Issue by stevejpurves** [ben-bougher/pickr#144]

**Comment by kwinkunks** [comment]
Yes, there is probably a better way... We added it so a person could delete an image they'd uploaded (until there is an interpretation on it). 

Also, we needed a way for admins to delete things (they can delete images, even with interpretations).

Part of the need came from a bug where, if a user aborted the upload process (say, by not submitting the upload form), the image stuck around (because it had already been uploaded) and showed up in the gallery view with no title, etc. So we needed a way to delete those. 

Or are you talking about smthg else, or has something changed to obviate these use cases? Definitely interested in simplifying!
